### DISCUSSION TITLE
Is there a way to disable response validation for a specific route?

### DISCUSSION THREAD

**Discussion by daniil-enplore** [luolingchun/flask-openapi3#104] [Q&A]
So I have an flask-openapi3 server. most of it serves json and that is amazing. However now I ran into the issue that I need to create a single special rout that serves txt/csv.

Is there a work around to disable validation for that route?

**Comment by luolingchun** [comment]
Response is not validated. Are you referring to request validation?

You can use `flask.request.data`, but that no OpenAI UI.

```python
from flask import request

@app.get("")
def get_data():
    request.data

```

**Reply by daniil-enplore** [reply]
thank you, that is exactly what i needed, 

it is understandable that there would not be any spec generated for such a rout, however it would be nice if there was a way to hand write open api spec for it and get it merged into the generated spec. 

is there a way to manually inject spec snippets into the auto generated spec?

**Reply by luolingchun** [reply]
I drafted a PR to discuss this issue, and I gave a solution that we can continue the discussion in the #109.

**Reply by daniil-enplore** [reply]
wow, it's impressive how fast you jumped on the ball!
